### ISSUE TITLE
Reverse DNS and reserved IP functionality

### ISSUE THREAD

**Issue by grinay** [squat/terraform-provider-vultr#34]
Hi. I'm just want to suggest add function  that i need. 
I need to have ability create reserved IP and also set reverse dns record for IPs.
Thank.

**Comment by grinay** [comment]
Hello. Absolutely right. This is what I need.

**Comment by rtizzy** [comment]
@squat 

Can you confirm how to utilize the reserved IP functionality? I'd like to attach one to a server for VPN functionality.

The quoted commit doesn't seem to be about that functionality

**Comment by rtizzy** [comment]
From what I can tell, this was the commit you're referring to 

https://github.com/squat/terraform-provider-vultr/commit/c8cc3a14027ffb531c7316f48ba5f624081372b1

**Comment by squat** [comment]
@rtizzy, very right. Thanks I just updated the comment.

With a reserved IP you can create an IP separately from your VPS and keep it even after you tear down your servers or other Vultr infra. This way you can tie your DNS to a known value and it will remain known.

If you want to have a reserved IP that survives infra churn, then you may want to have to separate terraform configurations. Otherwise you can do it all in one. The general idea is:
- create server
- create reserved_ip attached to server
- create DNS tied to reserved_ip.cidr
- use server and DNS record as you wish in your manifests

**Comment by squat** [comment]
Yes, this would also work. Reserved IPs were discussed in this thread simply because the original issue author said they required that feature.

**Comment by hielee** [comment]
Is there a timeline for when the Reverse DNS support will be shipped?
